pkg/fission-cli/cmd/environment/delete.go

```diff
@@ -43,7 +43,11 @@ func (opts *DeleteSubCommand) do(input cli.Input) error {
 
 	err := opts.Client().V1().Environment().Delete(m)
 	if err != nil {
-		return errors.Wrap(err, "error deleting environment")
+		if input.Bool(flagkey.EnvIgnoreNotFound) {
+			return nil
+		} else {
+			return errors.Wrap(err, "error deleting environment")
+		}
```

Comment:
This would improve readability.
```
if err != nil {
         if !input.Bool(flagkey.EnvIgnoreNotFound) {
	        return errors.Wrap(err, "error deleting environment")
         }
         return nil
}
```